app/services/idv/agent.rb

```diff
@@ -24,40 +24,14 @@ def proof_resolution(document_capture_session, should_proof_state_id:, trace_id:
           result_id: document_capture_session.result_id,
         )
       else
-        LambdaJobs::Runner.new(
-          job_class: Idv::Proofer.resolution_job_class,
-          in_process_config: {
-            aamva_config: {
-              auth_request_timeout: AppConfig.env.aamva_auth_request_timeout,
-              auth_url: AppConfig.env.aamva_auth_url,
-              cert_enabled: AppConfig.env.aamva_cert_enabled,
-              private_key: AppConfig.env.aamva_private_key,
-              public_key: AppConfig.env.aamva_public_key,
-              verification_request_timeout: AppConfig.env.aamva_verification_request_timeout,
-              verification_url: AppConfig.env.aamva_verification_url,
```

Comment:
we still need to pass these configs in, inside the job classes now right?

Reply:
nevermind, I didn't realize we had copied these there already

Reply:
yep, all good. we'll want to test in staging to make sure the credentials make it appropriately and everything though